README.md

```diff
@@ -41,8 +41,10 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
-```js
-// write code here
+```
+var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
+for (foods = "apple", "bananas", "strawberry";{
+  console:log("I want to eat a fruit")};)
```

Comment:
Would be helpful to review the syntax for a `for` [loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for).